### ISSUE TITLE
Loading SVG resource

### ISSUE THREAD

**Issue by alialbaali** [JetBrains/compose-jb#396]
I'm getting this error while trying to load an svg resource:

```kotlin
/home/alialbaali/.jdks/adopt-openjdk-11.0.10/bin/java: symbol lookup error: /home/alialbaali/.skiko/4e59a626ca6a32feb60e5f77c54f2abfdc8c0d3042af9680d7b00600deabcabd/libskiko-linux-x64.so: undefined symbol: _ZN5SkDOMC1Ev
```
Using the following code:
```kotlin
fun main() = Window {
    svgResource("Kotlin.svg")
}
```

I tried with both JDK 15 and 11 and the same error happens.

Compose version : 0.3.0-build154
Kotlin version : 1.4.30
OS: Pop!_OS 20.10

Kotlin.svg: 
```xml
<svg width="64" height="64" viewBox="0 0 64 64" fill="none" xmlns="http://www.w3.org/2000/svg">
    <path d="M0 64L32.1067 31.8933L64 64H0Z" fill="url(#paint0_linear)"/>
    <path d="M0 0H32.1067L0 34.6667V0Z" fill="url(#paint1_linear)"/>
    <path d="M32.1067 0L0 33.8133V64L32.1067 31.8933L64 0H32.1067Z" fill="url(#paint2_linear)"/>
    <defs>
        <linearGradient id="paint0_linear" x1="17.0234" y1="78.9486" x2="47.2606" y2="48.7113"
                        gradientUnits="userSpaceOnUse">
            <stop offset="0.09677" stop-color="#0095D5"/>
            <stop offset="0.3007" stop-color="#238AD9"/>
            <stop offset="0.6211" stop-color="#557BDE"/>
            <stop offset="0.8643" stop-color="#7472E2"/>
            <stop offset="1" stop-color="#806EE3"/>
        </linearGradient>
        <linearGradient id="paint1_linear" x1="4.48981" y1="12.863" x2="22.0516" y2="-4.69867"
                        gradientUnits="userSpaceOnUse">
            <stop offset="0.1183" stop-color="#0095D5"/>
            <stop offset="0.4178" stop-color="#3C83DC"/>
            <stop offset="0.6962" stop-color="#6D74E1"/>
            <stop offset="0.8333" stop-color="#806EE3"/>
        </linearGradient>
        <linearGradient id="paint2_linear" x1="-10.7751" y1="58.8414" x2="48.7803" y2="-0.714027"
                        gradientUnits="userSpaceOnUse">
            <stop offset="0.1075" stop-color="#C757BC"/>
            <stop offset="0.2138" stop-color="#D0609A"/>
            <stop offset="0.4254" stop-color="#E1725C"/>
            <stop offset="0.6048" stop-color="#EE7E2F"/>
            <stop offset="0.743" stop-color="#F58613"/>
            <stop offset="0.8232" stop-color="#F88909"/>
        </linearGradient>
    </defs>
</svg>
```

**Comment by olonho** [comment]
Fixed by https://github.com/JetBrains/skiko/commit/f2b5c39212177a36f07ddf10fc6e84ee46ebc899, shall be available soonish.

**Comment by olonho** [comment]
Fix is included in Skiko 0.2.12.

**Comment by olonho** [comment]
Must be fixed in 0.3.0-rc1.

**Comment by okushnikov** [comment]
Please check the <a href="https://youtrack.jetbrains.com/issue/CMP-5504">following ticket</a> on YouTrack for follow-ups to this issue. GitHub issues will be closed in the coming weeks.
